README.md

```diff
@@ -1406,11 +1406,11 @@ Fired when the battery level changes; sent no more frequently than once per minu
 #### Examples
 
 ```js
-import { useBatteryLevel } from 'react-native-device-info'
+import { useBatteryLevel } from 'react-native-device-info';
 
 const batteryLevel = useBatteryLevel(); // 0.759999
 
-<Text>{ batteryLevel }</Text>
+<Text>{batteryLevel}</Text>;
```

Comment:
I like the semi-colon additions but you've altered whitespace at the same time, always a no-no for drive-by contributions ;-)